connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -278,6 +283,14 @@ protected boolean hasState(KafkaServer server, Predicate<BrokerState> desiredSta
             return false;
         }
     }
+    
+    public boolean sslEnabled() {
+        final Object listeners = brokerConfig.get(KafkaConfig$.MODULE$.ListenersProp());
```

Comment:
`brokerConfig` is a `Properties`, so we can call `getProperty()` to get back a `String` directly